internal/wireguard/rule.go

```diff
@@ -13,6 +14,29 @@ func (w *Wireguard) addRule(rulePriority, firewallMark int) (
 	rule.Priority = rulePriority
 	rule.Mark = firewallMark
 	rule.Table = firewallMark
+	rule.Family = unix.AF_INET //nolint:all
+	if err := w.netlink.RuleAdd(rule); err != nil {
+		return nil, fmt.Errorf("cannot add rule %s: %w", rule, err)
+	}
+
+	cleanup = func() error {
+		err := w.netlink.RuleDel(rule)
+		if err != nil {
+			return fmt.Errorf("cannot delete rule %s: %w", rule, err)
+		}
+		return nil
+	}
+	return cleanup, nil
+}
+
+func (w *Wireguard) addRule6(rulePriority, firewallMark int) (
+	cleanup func() error, err error) {
+	rule := netlink.NewRule()
+	rule.Invert = true
+	rule.Priority = rulePriority
+	rule.Mark = firewallMark
+	rule.Table = firewallMark
+	rule.Family = unix.AF_INET6 //nolint:all
```

Comment:
Please specify the linters complaining with i.e. `//nolint:revive,nosnakecase` instead of using `all`